Products/SQLAlchemyDA/da.py

```diff
@@ -39,6 +39,44 @@
 }
 
 
+# Global registry of named SQLAlchemyDA instances
+_da_registry = {}
+
+
+def register_da(name, da_instance):
+    """
+    Register an SQLAlchemy database adapter by name as part of a module
+    level dict. This might be called early in Zope startup, so this type
+    of registration is necessary instead of a zope.component registration.
+    (The same reason z3c.sqlalchemy uses a module dict for registration)
+    """
+    global _da_registry
```

Comment:
The `global` is unneeded:  this code does not assign a new value to the global name, but rather mutates the existing dictionary in place.